sql/migrations/warehouse/000022_alter_wh_uploads_merged_schema.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE wh_uploads
+DROP COLUMN IF EXISTS mergedschema;
```

Comment:
Can we remove the column in a release/separate PR?

So we are able to revert back without manual intervention